Documentation/API/nft_the_world.yml

```diff
@@ -0,0 +1,513 @@
+swagger: "2.0"
+info:
+  description: "This is the API documentation for the NFT exchanging site"
+  version: "0.1.0"
+  title: "NFT the World!"
+  termsOfService: "http://localhost/"
+  contact:
+    url: "https://github.com/amosproj/amos2021ws07-nft-development"
+  license:
+    name: "MIT"
+    url: "https://spdx.org/licenses/MIT.html"
+host: "localhost"
+basePath: "/api/v1"
+schemes:
+- "https"
+paths:
+  /asset/{asset_name}:
+    get:
+      tags:
+        - "asset"
+      summary: "Browse item in category [asset_name]"
+      description: "Return a long list of items belonging to a queried category"
+      operationId: "getNFTbyAssetName"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "asset_category"
+        in: "path"
+        description: "Name of the asset category for which it should return the items"
+        required: true
+        type: "string"
+      responses:
+        "200":
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/PlaceHolder"
+        "400":
+          description: "Invalid ID supplied"
```

Comment:
What is the difference between invalid category identifier and not found? If there are invalid IDs, how is the defined naming pattern?